src/pages/index.js

```diff
@@ -55,41 +137,86 @@ const enableValidation = (config) => {
 
 enableValidation(formValidationOptions);
 
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (data) => {
-      cardSection.addItems(createCard({ name: data.name, link: data.link }));
-    },
-  },
-  selectors.cardSection
-);
+api
+  .getInitialCards()
+  .then((initialCards) => {
+    cardSection.renderItems(initialCards);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
 
 function handleImageClick(data) {
   cardPreviewImage.open(data);
 }
 
-cardSection.renderItems();
-
-const newcardPopup = new PopupWithForm(selectors.newCardModal, (cardData) => {
-  cardSection.addItems(createCard({ name: cardData.title, link: cardData.url }));
-  newcardPopup.close();
-  formValidators["card-form"].disableButton();
+//update avatar
+const avatarModal = new PopupWithForm("#modal-avatar", (userData) => {
+  const avatarSubmitButton = document.querySelector(
```

Comment:
It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine